rali/rali/source/rali_api_augmentation.cpp

```diff
@@ -1823,18 +1784,16 @@ raliCopy(
         RaliContext p_context,	
         RaliImage p_input,	
         bool is_output)	
-{	
-    Image* output = nullptr;	
+{
+    if(!p_context || !p_input)
+        THROW("Null values passed as input")
+    Image* output = nullptr;
     auto context = static_cast<Context*>(p_context);	
     auto input = static_cast<Image*>(p_input);	
     try	
     {	
-        if(!input || !context)	
-            THROW("Null values passed as input")	
-
-        output = context->master_graph->create_image(input->info(), is_output);	
-
-        context->master_graph->add_node<CopyNode>({input}, {output});	
+        output = context->master_graph->create_image(input->info(), is_output);
+        context->master_graph->add_node<CopyNode>({input}, {output});
```

Comment:
Codacy found an issue: [Either the condition '!p_context' is redundant or there is possible null pointer dereference: context.](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6114304)